Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md

```diff
@@ -0,0 +1,186 @@
+---
+post_title: How Can I Be Notified Any Time a Service Goes Down?
+username: farisnt@gmail.com
+Categories: PowerShell
+tags: PowerShell, WMI, Events
+Summary: Using Powershell to create temporary event monitoring using WMI
+
+---
+
+**How Can I Be Notified Any Time a Service Goes Down?**
+
+Q: How Can I Be Notified Any Time a Service Goes Down?
+
+A: The short quick answer to utilizing WMI and PowerShell 7.
+
+You use Powershell to create temporary event monitoring using WMI.
+Then WMI monitors any service changes and generates an alert once it detects a change.
+
+## Basic Requirement
+
+To achieve this, you need Windows PowerShell 5.1 and above.
+
+This post uses the latest version of PowerShell 7.
+So if you are don't yet have PowerShell 7, see the [Microsoft documentation on how to install PowerShell 7 on Windows.](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-7.1).

```

Comment:
When linking to docs, remove the locale (en-us) from the URL. Also remove them view parameter unless you are linking to a specific version of a cmdlet.

```suggestion
So if you are don't yet have PowerShell 7, see the Microsoft documentation on how to [Install PowerShell 7 on Windows](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-windows).
```